.github/workflows/coreaudio-sys.yml

```diff
@@ -37,6 +37,28 @@ jobs:
     - name: cargo doc - all features
       run: cargo doc --all-features --verbose
 
+  # Check that cross-compilation from linux is working.
+  # TODO: This is adapted from old travis file, should probably also:
+  # - Check more feature permutations.
+  # - Use more recent SDK? Looks like 10.13 is used atm.
+  # - Check iOS target?
+  linux-cross-compile:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install stable
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: stable
+        override: true
+    - name: Add apple target
+      run: rustup target add x86_64-apple-darwin
+    - name: Download SDK
+      run: curl -sL https://github.com/phracker/MacOSX-SDKs/releases/download/10.13/MacOSX10.13.sdk.tar.xz | tar -Jxf -; export COREAUDIO_SDK_PATH="$PWD/MacOSX10.13.sdk"
```

Comment:
How do we feel about maxing this a [`jobs.<job_id>.strategy.matrix`](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix) option and having a couple versions of the SDK being tested? One nice thing about linux -> macos cross compilation is that it's easy to get a bunch of SDKs and test against.

Reply:
Sounds good!

I guess one important thing to note is that if we want to make sure documentation builds successfully, we'll need to be able to recreate the same cross compilation approach (with same version targets etc) that docs.rs use, otherwise there's a chance the docs build will fail despite our CI passing. See https://github.com/rust-lang/docs.rs/issues/853.